soci/store/store.go

```diff
@@ -56,6 +56,11 @@ type Store interface {
 
 type ContentStoreType = config.ContentStoreType
 
+type ContentStoreConfig struct {
+	config.ContentStoreConfig
+	Namespace string
```

Comment:
Do we still need this here?

Reply:
Nope, this isn't wired to anything anymore. Got confused because we still need to attach to the namespace in the CLI but looks like we already have it covered [here](https://github.com/awslabs/soci-snapshotter/blob/fadf416a4d5120e340686e298b14688270a8551c/cmd/soci/commands/internal/client.go#L60)

Reply:
Thanks I was going to ask how the namespace is set for SOCI CLI create index. :)